src/etc/cargo.bashcomp.sh

```diff
@@ -246,7 +246,7 @@ _get_examples(){
 	local manifest=$(_locate_manifest)
 	[ -z "$manifest" ] && return 0
 
-	local files=("${manifest%/*}"/examples/*.rs)
+	local files=("${manifest%/*}"/examples/**/*.rs)
```

Comment:
Should we do something similar for zsh?
https://github.com/rust-lang/cargo/blob/c2c636a6dbe98f0b1a6f14ff5eefe2aa66a9243e/src/etc/_cargo#L472-L477